### PR TITLE
[graphicsmagick] update to version 1.3.37

### PR DESCRIPTION
**Pull request by josuegomes**
Update to version 1.3.37

### CODE REVIEW

File: ports/graphicsmagick/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "graphicsmagick",
-  "version-string": "1.3.36",
-  "port-version": 1,
+  "version-string": "1.3.37",
```
Comment (JonLiu1993) [inline ports/graphicsmagick/vcpkg.json:3]:
```suggestion
  "version": "1.3.37",
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/graphicsmagick/vcpkg.json (hunk 1 above)
Comment (josuegomes) [reply ports/graphicsmagick/vcpkg.json:3]: Is there a rationale for this change request?

Comment (JonLiu1993) [reply ports/graphicsmagick/vcpkg.json:3]: @josuegomes ,You can refer [this](https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#version)

Comment (josuegomes) [reply ports/graphicsmagick/vcpkg.json:3]: Thank you

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**